cmd/fs-v1.go

```diff
@@ -72,15 +72,131 @@ func initMetaVolumeFS(fsPath, fsUUID string) error {
 
 }
 
+// List all buckets at meta bucket prefix in `.minio.sys/buckets/` path.
+func readDirBuckets(metaBucket string) ([]string, error) {
```

Comment:
I think you can move this function to fs-v1-helpers.go, this function is already generic.